erigon-lib/state/domain.go

```diff
@@ -228,6 +229,10 @@ func (d *Domain) openFiles() (err error) {
 				continue
 			}
 			if item.decompressor, err = seg.NewDecompressor(datPath); err != nil {
+				d.logger.Debug("Domain.openFiles: %w, %s", err, datPath)
+				if errors.Is(err, &seg.ErrCompressedFileCorrupted{}) {
```

Comment:
usually into `errors.Is` need pass type `seg.ErrCompressedFileCorrupted`, not instance `&seg.ErrCompressedFileCorrupted{}`

Reply:
no, that looks like that because error is defined as `var ErrABC=  errors.new()`, while .Is takes instanced error as a argument)